R/equivalence_test.R

```diff
@@ -4,9 +4,9 @@
 #'
 #' @inheritParams rope
 #'
-#' @details Using the \link[=rope]{ROPE} and the \link[=hdi]{HDI}, Kruschke (2010, 2011, 2014, 2018) suggest using the percentage of the 95\% (or 90\%, considered more stable) \link[=hdi]{HDI} that falls within the ROPE as a decision rule. If the HDI is completely outside the ROPE, the "null hypothesis" for this parameter is "rejected". If the ROPE completely covers the HDI, i.e. all most credible values of a parameter are inside the region of practical equivalence, the null hypothesis is accepted. Else, it’s undecided whether to accept or reject the null hypothesis. If the full ROPE is used (i.e., 100\% of the HDI), then the null hypothesis is rejected or accepted if the percentage of the posterior within the ROPE is smaller than to 1\% or greater than 99\%.
+#' @details Using the \link[=rope]{ROPE} and the \link[=hdi]{HDI}, Kruschke (2018) suggest using the percentage of the 95\% (or 90\%, considered more stable) \link[=hdi]{HDI} that falls within the ROPE as a decision rule. If the HDI is completely outside the ROPE, the "null hypothesis" for this parameter is "rejected". If the ROPE completely covers the HDI, i.e. all most credible values of a parameter are inside the region of practical equivalence, the null hypothesis is accepted. Else, it’s undecided whether to accept or reject the null hypothesis. If the full ROPE is used (i.e., 100\% of the HDI), then the null hypothesis is rejected or accepted if the percentage of the posterior within the ROPE is smaller than to 1\% or greater than 99\%.
 #'
-#' @references Kruschke JK. Rejecting or Accepting Parameter Values in Bayesian Estimation. Advances in Methods and Practices in Psychological Science. 2018; doi: \doi{10.1177/2515245918771304}
+#' @references Kruschke JK. Rejecting or Accepting Parameter Values in Bayesian Estimation. Advances in Methods and Practices in Psychological Science. 2018; \doi{10.1177/2515245918771304}
```

Comment:
Should we standardize citation style  (APA norms, ...)?

Reply:
Yes, we probably could use a reference manager (I personally use zotero) to create the references or so.

Reply:
Even faster could be to use the **cite** button below the paper on [google scholar](https://scholar.google.fr/scholar?hl=fr&as_sdt=0%2C5&q=Kruschke+JK.+Rejecting+or+Accepting+Parameter+Values+in+Bayesian+Estimation.+Advances+in+Methods+and+Practices+in+Psychological+Science.+2018&btnG=) and then copy paste the APA reference

Reply:
I have those citations in zotero anyway, so it's also just a click away

Reply:
APA is ok for my perspective (guess I am used to it), but as long as it's consistent...